app/views/group/show.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="group_container_show">
+<div class="title_show">
+<h1><%= @group.name %></h1>
+<p>$<%= @total %></p>
+</div>
+<% @group_items.each do |item| %>
+<div class="group">
+<div class="amount-name">
+<h3><%= item.name%></h3>
+<p>$<%= item.amount %></p>
+</div>
+<div class="group_item">
+<p><%= item.icon %></p>
+<p class="show_date"><%= item.created_at.strftime('%A, %B %d, %Y') %></p>
+</div>
+</div>
+<% end %>
+<%= link_to 'View Items', item_path(@group), class: 'view_link' %>
+</div>
+
+  <div class="add_back_container">
+<div class="add_new">
+<%= link_to 'Back to Group', group_index_path, class: 'back_to_group' %>
```

Comment:
- [ ] Like the previous reviewer mentioned, please make sure this button enables a user create an item in a category. Currently, it just takes me back to the group page. Please fix this.